examples/trading/frontend/src/hooks/useTransactionExecution.ts

```diff
@@ -1,8 +1,8 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
-import { useSignTransactionBlock, useSuiClient } from "@mysten/dapp-kit";
+import { useSignTransaction, useSuiClient } from "@mysten/dapp-kit";
 import { SuiTransactionBlockResponse } from "@mysten/sui/client";
-import { TransactionBlock } from "@mysten/sui/transactions";
+import { Transaction} from "@mysten/sui/transactions";
```

Comment:
I use VS Code and a plugin I installed at some point fixes these lints for me. I think you can run `pnpm run lint:fix` to auto correct them.